startVM.mjs

```diff
@@ -0,0 +1,49 @@
+/**
+ * Démarre la VM devbox en utilisant VBoxManage plutôt que vagrant up
+ */
+
+import { execSync } from 'node:child_process'
+import { devboxFolder, devboxSsh, devboxVMNamePrefix, vBoxManageExe } from './variables.mjs'
+
+// Use first argument or find the matching vm from VBoxManage list vms
+let vmIdentifier = null;
+if (process.argv.length > 2) {
+    vmIdentifier = process.argv[2];
+} else {
+    // Gets the list of all VMS in the virtualbox
+    const vms = execSync(`"${vBoxManageExe}" list vms`, { encoding: 'utf8' })
+    const vmInfos = vms.split('\n').map(vm => {
+        const match = vm.match(/"(.+)" {(.+)}/)
+        if(match) {
+            return {
+                name: match[1],
+                uuid: match[2]
+            }
+        }
+        return null
+    }).filter(vm => vm !== null);
+    
+    const foundVm = vmInfos.find(vm => vm.name.startsWith(devboxVMNamePrefix))

```

Comment:
We need to take into account that a person can have 2 devbox or VM with the same prefix.

Reply:
It's up to you to configure the prefix properly. If the default prefix doesn't fit your use case, include more characters, or maybe the whole name. The only way this can fail is if you have two VM 'X' and 'X2', but this doesn't happen with vagrant.